index.html

```diff
@@ -45,6 +45,224 @@ <h1 class="name">I’m Ali Hassan. Glad to see you!</h1>
+              </li>
+              <li>
+                <a class="github" href="#"
+                  ><img src="./images/main/Lomri.svg" alt="github"
+                /></a>
+              </li>
+              <li>
+                <a class="V" href="#"
+                  ><img src="./images/main/V.svg" alt="V"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <section id="works-section">
+        <div class="work-projects">
+          <div class="item project-header">
+            <div class="project-img-container">
+              <img src="./images/work/img_1.PNG" alt="" />
+            </div>
+          </div>
+          <div class="card-content">
+            <div class="item project-description">
+              <div class="project-title-container">
+                <h4 class="description-title">Tonic</h4>
+              </div>
+              <div class="project-subtitle-container">
+                <ul class="subtitle-items">
+                  <li class="list-less"><p>CANOPY</p></li>
+                  <li class="list"><p>Back End Dev</p></li>
+                  <li class="list"><p>2015</p></li>
+                </ul>
+              </div>
+              <div class="project-article-container">
+                <p class="description-item description-item-b">
+                  A daily selection of privately perosonalized reads; no
+                  accounts or sign-ups required.
+                </p>
+              </div>
+              <div class="project-tag-container">
+                <ul class="tag-item">
+                  <li class="tags"><a class="tag-btn" href="#">html</a></li>
+                  <li class="tags"><a class="tag-btn" href="#">css</a></li>
+                  <li class="tags">
+                    <a class="tag-btn" href="#">javaScript</a>
+                  </li>
+                </ul>
+              </div>
+              <div class="project-button-container">
+                <a href="#" class="project-button">See project</a>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="work-projects">
+          <div class="item project-header">
+            <div class="project-img-container_1">
+              <img src="./images/work/img_2.PNG" alt="" />
+            </div>
+          </div>
+          <div class="card-content">
+            <div class="item project-description">
+              <div class="project-title-container">
+                <h4 class="description-title">
+                  Multi-Post <br />
+                  Stories
+                </h4>
+              </div>
+              <div class="project-subtitle-container">
+                <ul class="subtitle-items">
+                  <li class="list-less"><p>CANOPY</p></li>
+                  <li class="list"><p>Back End Dev</p></li>
+                  <li class="list"><p>2015</p></li>
+                </ul>
+              </div>
```

Comment:
Kindly update the styling of each of your cards to match the figma design better🙏

Your design:
<img width="237" alt="image" src="https://github.com/mhassa204/Portfolio-Project/assets/37116322/e8e3d4aa-459c-4d27-a5c6-fb3b75394b89">

Figma design:
<img width="219" alt="image" src="https://github.com/mhassa204/Portfolio-Project/assets/37116322/7f859ec0-008e-4473-8de0-04480e1c36b0">

A few things to note:
- Please increase the border radius to 16px
- Kindly align the items of the sublist under the title
- Kindly align the list of technologies to the left
- Kindly make sure the image is properly centered